enclave/src/main/java/tech/pegasys/pantheon/enclave/Enclave.java

```diff
@@ -76,8 +78,19 @@ private Request buildPostRequest(
     try (Response response = client.newCall(request).execute()) {
       return objectMapper.readValue(response.body().string(), responseType);
     } catch (IOException e) {
-      LOG.error("Enclave failed to execute ", request);
-      throw new IOException("Enclave failed to execute post", e);
+      LOG.error("Enclave failed to execute {}", request);
+      String regex = "\\{(.*?)(\"([^\"]*)\")\\}";
```

Comment:
Is there any way to avoid regex? Ideally, the response should be serialized to an object.